src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/data/CreateRoomMappingDto.kt

```diff
@@ -13,6 +13,6 @@ data class CreateRoomMappingDto(
   @Schema(description = "nomis room description (unique within prison)", required = true)
   val nomisRoomDescription: String,
 
-  @Schema(description = "open or closed indicator", required = true)
-  val isOpen: Boolean,
+  @Schema(description = "open or closed indicator, default is false")
+  val isOpen: Boolean = false,
```

Comment:
Not expecting any functional changes - though I guess this is non-breaking.

Reply:
agreed, wasn't expecting to see that either.  I'll check to see what has gone wrong

Reply:
Ah it's opk - there was a fix from Lucy in the middle of the commits